### PR TITLE
VS SDK update to 16.1

### PR DESCRIPTION
**Pull request by ivanbasov**
Need it for the right fix of https://github.com/dotnet/roslyn/issues/34089

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -299,6 +299,7 @@
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
       https://dotnet.myget.org/F/dotnet-corefxtestdata/api/v3/index.json;
       https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
+      https://dotnet.myget.org/F/experimental-vs-packages/api/v3/index.json;
```
Comment (jasonmalinowski) [inline eng/Versions.props]: Why the new feed?

Comment (ivanbasov) [reply eng/Versions.props]: @jasonmalinowski this is the best we found we @heejaechang . Please let me know if we can use an existing one. Which one?

Comment (jasonmalinowski) [reply eng/Versions.props]: Why aren't these in the editor feed? If it can't be there then we can put it in roslyn-tools.

Comment (ivanbasov) [reply eng/Versions.props]: It is not directly made by Editor. It is Microsoft.VisualStudio.Utilities. See the email thread.
All Editor binaries are properly published in existing Editor feeds.

Comment (jasonmalinowski) [reply eng/Versions.props]: Ah, then this should be in roslyn-tools.

Comment (jasonmalinowski) [reply eng/Versions.props]: For those following along at home, I uploaded the package.

**Pull request merged**